src/ert/gui/tools/plot/customize/customize_plot_dialog.py

```diff
@@ -184,24 +190,32 @@ class CustomizePlotDialog(QDialog):
     copySettings = Signal(str)
     copySettingsToOthers = Signal(list)
 
-    def __init__(self, title, parent, key_defs: List[PlotApiKeyDefinition], key=""):
+    def __init__(
+        self,
+        title: Optional[str],
+        parent: Optional[QWidget],
+        key_defs: List[PlotApiKeyDefinition],
+        key: Optional[str] = "",
+    ) -> None:
         QDialog.__init__(self, parent)
         self.setWindowTitle(title)
 
         self.current_key = key
         self._key_defs = key_defs
 
-        self.setWindowFlags(self.windowFlags() & ~Qt.WindowContextHelpButtonHint)
-        self.setWindowFlags(self.windowFlags() & ~Qt.WindowCloseButtonHint)
+        self.setWindowFlags(self.windowFlags() & ~Qt.WindowContextHelpButtonHint)  # type: ignore
+        self.setWindowFlags(self.windowFlags() & ~Qt.WindowCloseButtonHint)  # type: ignore
 
-        self._tab_map = {}
-        self._tab_order = []
+        self._tab_map: dict[str, QWidget] = {}
+        self._tab_order: list[str] = []
 
         layout = QVBoxLayout()
 
         self._tabs = QTabWidget()
         layout.addWidget(self._tabs)
-        layout.setSizeConstraint(QLayout.SetFixedSize)  # not resizable!!!
+        layout.setSizeConstraint(
+            QLayout.SizeConstraint.SetFixedSize
+        )  # not resizable!!!
```

Comment:
This seems like a pretty pointless comment

Reply:
removed